### ISSUE TITLE
Question: Is the premium pro account per org or for all orgs in an account?

### ISSUE THREAD

**Issue by algoflows** [denoland/deploy_feedback#448]
### What problem are you trying to solve?

Hi, its not clear to if users get access to all orgs in an account if we pay the 20 bucks per person? We are a team that work across multiple orgs and it wouldn't work if we need to pay for user access to each organisation....

### Describe the solution you'd like

na

### Describe alternatives you've considered

na

### Documentation, Adoption, Migration Strategy

na

**Comment by alonbonder** [comment]
Hi Sean, thanks for reaching out. Would you be open to a quick chat about this live? I'd like to understand your goals in setting up multiple orgs per account. The per-member pricing is actually something we're still thinking through. Feel free to shoot me a note at alon@deno.com if that works. 

-Alon

**Comment by algoflows** [comment]
Hey Alon,

My team often collaborates across multiple organisations within a single account. The reason for this is the diversity of projects we're handling simultaneously and the flexibility it offers us in managing our resources.

In my experience with other platforms, one pain point we've often encountered is the additional cost incurred when a single user is part of multiple organisations or projects within the same account. This kind of double billing isn't ideal for us and can sometimes discourage our team members from engaging fully with all the organisations they're a part of.

It would be immensely beneficial for us if your platform tied billing to the user account or top level admin account, rather than the number of organisations they join. This means a user could contribute to multiple organisations without incurring extra costs. We appreciate that you're still thinking through the per-member pricing, and we'd highly recommend considering this approach.

A platform that's got this right, in my opinion, is GitHub. They've implemented a model where users can belong to multiple projects/repo and teams without extra charges. This kind of flexibility in terms of payment structure and resource management provides a more user-friendly experience.

I'll reach out via email.

Best,

Sean.

**Comment by beingminimal** [comment]
> Hey Alon,
> 
> My team often collaborates across multiple organisations within a single account. The reason for this is the diversity of projects we're handling simultaneously and the flexibility it offers us in managing our resources.
> 
> In my experience with other platforms, one pain point we've often encountered is the additional cost incurred when a single user is part of multiple organisations or projects within the same account. This kind of double billing isn't ideal for us and can sometimes discourage our team members from engaging fully with all the organisations they're a part of.
> 
> It would be immensely beneficial for us if your platform tied billing to the user account or top level admin account, rather than the number of organisations they join. This means a user could contribute to multiple organisations without incurring extra costs. We appreciate that you're still thinking through the per-member pricing, and we'd highly recommend considering this approach.
> 
> A platform that's got this right, in my opinion, is GitHub. They've implemented a model where users can belong to multiple projects/repo and teams without extra charges. This kind of flexibility in terms of payment structure and resource management provides a more user-friendly experience.
> 
> I'll reach out via email.
> 
> Best,
> 
> Sean.

@alonbonder @algoflows Agree...this should be as suggested here. Waiting for this feature. Thanks
